cost-analyzer/templates/_helpers.tpl

```diff
@@ -1061,9 +1072,18 @@ Begin Kubecost 2.0 templates
     - name: actions-config
       mountPath: /var/configs/actions
     {{- end }}
-    {{- if or ((.Values.kubecostProductConfigs).actions).storageConfigSecret ((.Values.kubecostProductConfigs).actions).storageConfig }}
+    {{- if ((.Values.kubecostProductConfigs).actions).enabled }}
+    {{- if not (eq ((.Values.kubecostProductConfigs).actions).storageConfigSecret (.Values.kubecostModel).federatedStorageConfigSecret) }}
     - name: actions-storage-config
       mountPath: /var/configs/actions/storage
+    {{- else }}
+    - name: actions-storage
+      mountPath: /var/configs/actions/storage
```

Comment:
Having the `emptyDir` mounted regardless (ie. under the `enabled` block) seems to clash with the actions-storage-config volume. I saw an empty `storage` dir when testing. This works though.